include/ed/plugin_container.h

```diff
@@ -61,14 +61,10 @@ class PluginContainer
         world_new_ = world;
     }
 
-    void setLoopFrequency(double freq) { loop_frequency_ = freq; }
+    void setLoopFrequency(double freq) { loop_frequency_ = freq; loop_frequency_max_ = 1.05*freq; loop_frequency_min_= 0.9*freq; }
```

Comment:
Magic numbers. Please find a way to make these more configurable. Or at least make the margins explicit in the diagnostics message.

Or: add setter functions for max and min frequencies

Reply:
The `LoopUsageStatus` doesn't contain any hardcode stuff/magic number. So adding these magic numbers in there is only making the code dirtier.
I am open to adding the functionality of frequency bounds at a later moment. For now I just add the magic number comment.

This is diagnostics, so we these numbers are not that important IMHO.

Reply:
Add the magic number comment